aten/src/ATen/native/Sorting.cpp

```diff
@@ -519,8 +518,7 @@ Tensor& quantile_out(
       out,
       self,
       q,
-      // NOLINTNEXTLINE(performance-move-const-arg)
-      std::move(dim),
+      dim,
```

Comment:
Why change this from a move?

Reply:
dim is trivially copyable here so std::move has no effect and removing it also cleans up the NOLINT comments.